### DISCUSSION TITLE
Anchor Links

### DISCUSSION THREAD

**Discussion by alexkuzmin** [craftcms/ckeditor#169] [Ideas] [answered]
For the successful application of the WCAG 2.1 standard guidelines as outlined here https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html, there's a requirement for the capability to establish anchor links. Currently, this functionality is not available within the CKEditor plugin for CraftCMS. However this feature has been successfully integrated into the Redactor Plugin: https://plugins.craftcms.com/redactor-anchors?craft4=. Considering this, one might wonder if a similar implementation could be feasible for the CKEditor plugin, too.

**Comment by brandonkelly** [comment, marked as answer]
CKEditor5 doesn’t have a first party package for creating anchors ([yet](https://github.com/ckeditor/ckeditor5/issues/1944)), but it looks like [ckeditor5-anchor-drupal](https://github.com/northernco/ckeditor5-anchor-drupal) has become the de facto way to do it in the meantime. So we just released CKEditor 3.8.0 with that plugin added.
